apps/amiapp_flutter/ios/Podfile

```diff
@@ -52,8 +52,9 @@ end
 
 target 'NotificationServiceExtension' do
   use_frameworks!
-  # Uncomment only 1 of the lines below to install a version of the iOS SDK
-  pod 'CustomerIO/MessagingPushFCM', '~> 3.5' # install production build
+  # Ideally, installing non-production SDK to main target should be enough
+  # We may not need to install non-production SDK to app extension separately
```

Comment:
interesting, how would installing dependency in the main target be enough for NSE as well?

Reply:
From my understanding, it refers to local Pods, which will be already installed by main target, so there's no need to reinstall local pods separately.